frb_codegen/src/markers.rs

```diff
@@ -1,7 +1,7 @@
 use syn::*;
 
 /// Extract a path from marker `#[frb(mirror(path), ..)]`
-pub fn extract_mirror_marker(attrs: &[Attribute]) -> Option<Path> {
+pub fn extract_mirror_marker(attrs: &[Attribute]) -> Vec<Path> {
```

Comment:
Please add a bit of test in `pure_dart` api.rs and main.dart, so that we can ensure it works now and works in the future :)

(unrelated to this line of code; just b/c no place to put this comment)